arviz/data/io_cmdstan.py

```diff
@@ -0,0 +1,417 @@
+"""CmdStan-specific conversion code."""
+from collections import defaultdict
+from copy import deepcopy
+from glob import glob
+import linecache
+import os
+import re
+
+
+import numpy as np
+import pandas as pd
+import xarray as xr
+
+from .inference_data import InferenceData
+from .base import requires, dict_to_dataset, generate_dims_coords
+
+
+class CmdStanConverter:
+    """Encapsulate CmdStan specific logic."""
+
+    # pylint: disable=too-many-instance-attributes
+
+    def __init__(self, *, output=None, prior=None, posterior_predictive=None,
+                 observed_data=None, observed_data_var=None,
+                 log_likelihood=None, coords=None, dims=None):
+        self.output = glob(output) if isinstance(output, str) else output
+        if isinstance(output, str) and len(self.output) > 1:
+            msg = "\n".join("{}: {}".format(i, os.path.normpath(path)) \
+                            for i, path in enumerate(self.output, 1))
+            print("glob found {} files for 'output':\n{}".format(len(self.output), msg))
+        if isinstance(prior, str) and prior.endswith(".csv"):
+            prior = glob(prior)
+            if len(prior) > 1:
+                msg = "\n".join("{}: {}".format(i, os.path.normpath(path)) \
+                                for i, path in enumerate(prior, 1))
+                print("glob found {} files for 'prior':\n{}".format(len(prior), msg))
+        self.prior = prior
+        self.posterior_predictive = posterior_predictive
+        self.observed_data = observed_data
+        self.observed_data_var = observed_data_var
+        self.log_likelihood = log_likelihood
+        self.coords = coords if coords is not None else {}
+        self.dims = dims if dims is not None else {}
+        self.chains = len(output)
```

Comment:
this variable does not look like it is used anywhere else

Reply:
Oh, it might be true. Also it could be wrong with future changes.